operator/v1/0000_50_ingress-operator_00-ingresscontroller.crd.yaml

```diff
@@ -1214,18 +1214,22 @@ spec:
                       waits between two consecutive health checks on its configured
                       backends.  This value is applied globally as a default for all
                       routes, but may be overridden per-route by the route annotation
-                      \"router.openshift.io/haproxy.health.check.interval\". \n Setting
-                      this to less than 5s can cause excess traffic due to too frequent
-                      TCP health checks and accompanying SYN packet storms.  Alternatively,
-                      setting this too high can result in increased latency, due to
-                      backend servers that are no longer available, but haven't yet
-                      been detected as such. \n An empty or zero healthCheckInterval
-                      means no opinion and IngressController chooses a default, which
-                      is subject to change over time. Currently the default healthCheckInterval
-                      value is 5s. \n Currently the minimum allowed value is 1s and
-                      the maximum allowed value is 2147483647ms (24.85 days).  Both
-                      are subject to change over time."
-                    format: duration
+                      \"router.openshift.io/haproxy.health.check.interval\". \n Expects
+                      an unsigned duration string of decimal numbers, each with optional
+                      fraction and a unit suffix, eg \"300ms\", \"1.5h\" or \"2h45m\".
+                      Valid time units are \"ns\", \"us\" (or \"µs\"), \"ms\", \"s\",
+                      \"m\", \"h\". \n Setting this to less than 5s can cause excess
+                      traffic due to too frequent TCP health checks and accompanying
+                      SYN packet storms.  Alternatively, setting this too high can
+                      result in increased latency, due to backend servers that are
+                      no longer available, but haven't yet been detected as such.
+                      \n An empty or zero healthCheckInterval means no opinion and
+                      IngressController chooses a default, which is subject to change
+                      over time. Currently the default healthCheckInterval value is
+                      5s. \n Currently the minimum allowed value is 1s and the maximum
+                      allowed value is 2147483647ms (24.85 days).  Both are subject
+                      to change over time."
+                    pattern: ^0|([0-9]+(\.[0-9]+)?(ns|us|µs|ms|s|m|h))+$
```

Comment:
The `time` package accepts both U+00B5 (micro symbol) and U+03BC (Greek letter mu) :upside_down_face:.

Reply:
It's more in line with the godoc for https://pkg.go.dev/time#ParseDuration, but I can change it.  We have the only the micro symbol everywhere else we use this pattern, and might not fix a bug where someone wanted to use a mu rather than a micron, but I will.